src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -84,9 +87,17 @@ const DataIntegration = () => {
   const router = useRouter();
   const { experimentId } = router.query;
 
+  const calculationConfig = useSelector((state) => state.experimentSettings.processing.dataIntegration);
+
   const [activePlotKey, setActivePlotKey] = useState('elbowPlot');
   const [config, setCurrentConfig] = useState(persistedConfigs.elbowPlot);
 
+  useEffect(() => {
+    if (experimentId && !calculationConfig) {
+      loadProcessingSettings(experimentId);
```

Comment:
actually, you should use the imported `dispatch` in here

Reply:
Yea, I missed that. Added in next commit.